api/src/main/java/org/pmiops/workbench/cohortreview/CohortReviewServiceImpl.java

```diff
@@ -134,7 +143,42 @@ public ParticipantCohortStatus findParticipantCohortStatus(Long cohortReviewId,
     }
 
     @Override
-    public ParticipantCohortAnnotation saveParticipantCohortAnnotation(ParticipantCohortAnnotation participantCohortAnnotation) {
+    public ParticipantCohortAnnotation saveParticipantCohortAnnotation(Long cohortReviewId, ParticipantCohortAnnotation participantCohortAnnotation) {
```

Comment:
Add some tests for adding annotations of different types (with good or bad values for enums and dates)?